### ISSUE TITLE
Scheduled Notification Plugin should render the configured templates

### ISSUE THREAD

**Issue by jmbrunskill** [openmsupply/notify#136]
## Is your feature request related to a problem? Please describe

Following on from #125 when processing scheduled notifications, the code should attempt to render the correct template with configured parameters.

**Comment by nisha-dangol** [comment]
Testing currently in [test-notify.msupply.org](https://test-notify.msupply.org/)

Telegram notification does look good:
<img width="329" alt="image" src="https://github.com/openmsupply/notify/assets/59193490/40a70d77-9169-4603-a605-916eb456cdb6">

..but maybe it'd be good if those details fields (`Subject template`, `Body template`, `Parameters`) could also look wide (like that `Notification Title` field) as in [your screenshot](https://github.com/openmsupply/notify/pull/156#issue-1916587553) @jmbrunskill 
<img width="1370" alt="image" src="https://github.com/openmsupply/notify/assets/59193490/58898bd6-fb17-487b-be79-3b619202c2da">

**Comment by jmbrunskill** [comment]
@nisha-dangol Yes definitely should be wider, thanks for letting me know. Not sure what happened but I'll look into it!
